### PR TITLE
go.mod: require go1.19

### PR DESCRIPTION
**Pull request by johanneskastl**
I am packaging this nice plugin for openSUSE and noticed the following errors:

```
[    5s] + go build -mod=vendor -buildmode=pie '-ldflags=-X main.Version=0.1.9' -o bin/community_images ./cmd/community_images/
[    9s] # k8s.io/apimachinery/pkg/util/sets
[    9s] vendor/k8s.io/apimachinery/pkg/util/sets/byte.go:34:17: type parameter requires go1.18 or later (-lang was set to go1.16; check go.mod)
[...]
[    9s] vendor/k8s.io/apimachinery/pkg/util/sets/set.go:182:29: too many errors
[   15s] # sigs.k8s.io/json/internal/golang/encoding/json
[   15s] vendor/sigs.k8s.io/json/internal/golang/encoding/json/encode.go:158:16: predeclared any requires go1.18 or later (-lang was set to go1.16; check go.mod)
[...]
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module github.com/dims/community-images
 
-go 1.15
+go 1.18
```
Comment (nikhita) [inline go.mod:3]: Can we use go 1.19 instead?

Comment (johanneskastl) [reply go.mod:3]: If desired I can of course use go1.19. I only took the minimal version that might solve the errors.

**Comment by johanneskastl** [comment]
OK, I redid the last commit, so it actually uses go1.19.

**Comment by dims** [comment]
@johanneskastl can you please rebase so the new github action from @nikhita runs and gives us a thumbs up?

**Comment by johanneskastl** [comment]
> @johanneskastl can you please rebase so the new github action from @nikhita runs and gives us a thumbs up?

Done.

**Comment by dims** [comment]
going to land this now without the GH action triggering. we can watch that in other PRs as they flow in.

**Pull request merged**